nx.json

```diff
@@ -88,28 +88,27 @@
     }
   },
   "release": {
-    "projects": ["packages/*", "!packages/nx-plugin"],
+    "projects": ["packages/*"],
```

Comment:
I enabled the release of the `nx-plugin` project, so it is published during the initialization of the local registry  (then it can be installed in `global-setup.e2e.ts`).
Is it actually ready to be published on NPM?

Reply:
We need to have docs cleaned then the MVP would already work... 

So no.

Reply:
All right, i reverted the change and removed steps for publishing and installing the `nx-plugin` in `global-setup.e2e.ts`.